providers.go

```diff
@@ -195,3 +225,10 @@ func envProvider(s *Store, prefix string) {
 		}
 	}
 }
+
+func buildProviderName(name string, refresh bool, parameter string) string {
+	if refresh {
+		name = fmt.Sprintf("%s_refresh", name)
```

Comment:
Just return directly the return of `fmt.Sprintf`, we'll save one alloc, and it's easier to read.